.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,7 @@
 >^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+> Please delete this note before submitting the pull request.
 > For 1.12 Features: set Milestone to 1.12 and Base Branch to release-1.12
```

Comment:
Do we have instructions somewhere that provides an example on how to set the base branch to release-1.12?  For folks new to git that would be very helpful.  Also do we have instructions somewhere on setting the milestone. People will ask how to do that as well